docs/readmes/orc8r/deploy_build.md

```diff
@@ -76,32 +83,31 @@ We'll build the Orchestrator Helm charts, as well as publish them to a
 To start, create a private GitHub repo to use as your Helm chart repo. We'll
 refer to this as `GITHUB_REPO`.
 
-Next, package the Magma Helm charts and publish them to the GitHub
-repo. You'll have to check out a temporary commit to build the `1.4.36` version
-of the orc8r chart for 1.3.x because `1.4.37` on the head of the release
-branch has some changes that you shouldn't try to deploy yet.
+Next, define some necessary variables
 
 ```bash
-cd MAGMA_ROOT
-git checkout a7580153
+export GITHUB_REPO=<github_repo>
+export GITHUB_REPO_URL=<github_repo_url>
+export GITHUB_USERNAME=<github_username>
+export GITHUB_ACCESS_TOKEN=<github_access_token>
+```
 
-mkdir ~/magma-charts && cd ~/magma-charts
-git init
-helm package MAGMA_ROOT/orc8r/cloud/helm/orc8r/ && helm repo index .
-git add . && git commit -m 'Initial chart commit'
-git remote add origin GITHUB_REPO_URL && git push -u origin master
+Now run the package script. This script will package and publish the necessary
+helm charts to the `GITHUB_REPO`. The script expects a deployment type to be
+provided. The valid options are:
+* `fwa`
+* `federated_fwa`
+* `all`
 
+This variable specifies which orc8r modules will be deployed.
+To run the script:
+
+```bash
+$MAGMA_ROOT/orc8r/tools/helm/package.sh -d <deployment_type>
 ```
 
-To confirm, reference the published charts locally
+If successful, the script will output:
```

Comment:
Nit: I generally prefer *not* including colons here, just cause it's unnecessary